var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -90,6 +95,9 @@ class Graphviz(AutotoolsPackage):
     for b in tested_bindings + untested_bindings:
         depends_on('swig', when=b)
 
+    depends_on('cairo~X', when='+pangocairo')
+    depends_on('pango~X', when='+pangocairo')
```

Comment:
What's the emoticon for "opening a can of worms"?

I'll see what happens with the `+X` variants (seems like I lose a day every time I try this...).